src/App.js

```diff
@@ -4,9 +4,23 @@ import Question from './Question.js';
 import data from'./Data.js';
 
 export default class App extends Component {
-  render() {
+  constructor() {
+    super(...arguments);
+
+    this.onStartQuiz = this.onStartQuiz.bind(this);
+
+    this.state = { startQuiz: false };
```

Comment:
is it possible to set this up with redux and dispatch an action? so it follows the patter used by the Question Component?